### PR TITLE
30 card files link

### LINKED ISSUE #30: Component - Card_Files_Link
**Widget**: Card_Files_Link

**Image**:
Card_Files_Link

**Dimensions/Styles**: https://zpl.io/Vk0w7YL

**Inputs**
- String: Description of file/link navigating to
[Requires explanation from client]

**Notes**: (Requires validation)
- Navigates to external resources: Files, Link...
- The description should be only 1 line

### PR DESCRIPTION
**Pull request by CodeGlitch**
Widget proposal for Card_Files_Link
Accepts text for display and link to launch url.
(changed img, had uploaded the initial wrong one)

Color is expected to come from theme
close #30

### CODE REVIEW

File: lib/ui/widgets/card_files_link.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        child: Card(
+          color: Colors.white,
+          child: Padding(
+            padding: const EdgeInsets.all(8),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: <Widget>[
+                Expanded(
+                  child: Text(
+                    text.length > 15 ? text.substring(0, 15) + "..." : text,
```
Comment (miguelpruivo) [inline lib/ui/widgets/card_files_link.dart]: I'm not sure if this logic should stay here. The widget's text logic should be defined before-hand.

**Comment by miguelpruivo** [comment]
@CodeGlitch also, don't forget to rebase before pushing.

File: lib/ui/widgets/card_files_link.dart (hunk 1 above)
Comment (CodeGlitch) [reply lib/ui/widgets/card_files_link.dart]: Removed logic on 0bc1a16

**Comment by miguelpruivo** [comment]
LGTM. Fixed your conflicts. Waiting for CI to merge.

**Pull request merged**